### PR TITLE
Increase coverage by adding test for searchbar  

### PR DESCRIPTION
**Pull request by monikaprabhu**
Add tests to increase coverage for Searchbar.
- Test to search an item and verify the correct results are returned. 
- Test to change the view to detail view and make sure search  display item description for each item in the result. 
 

- Update the version of Jest to fix the mapping of line of code covered.
Before 
image

After
image
  
Coverage before the test were added : 
image

Coverage after the tests :    
image

- [x] The code base is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions), and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.

### CODE REVIEW

**Comment by monikaprabhu** [comment]
@fanwgwg  , @Benglin  please review and merge the tests

File: package.json [hunk 1]
```diff
@@ -15,33 +15,33 @@
   "author": "Dynamo Team",
   "license": "MIT",
   "dependencies": {
+    "@types/jest": "^20.0.0",
+    "babel-jest": "^20.0.3",
     "express": "^4.14.0",
+    "jest": "^20.0.4",
```
Comment (fanwgwg) [inline package.json]: Can we put these few dependencies to `devDependencies`, which means we should install these dependencies by commands like `npm install jest --save-dev`

**Comment by monikaprabhu** [comment]
@fanwgwg  please review

**Comment by fanwgwg** [comment]
@monikaprabhu This looks good to me!

**Pull request merged**